### PR TITLE
[LLM] Use to build a more slim docker for k8s

### PR DESCRIPTION
**Pull request by Romanticoseu**
## Description

Use to build a more slim docker for k8s

### CODE REVIEW

**Comment by Romanticoseu** [comment]
> No change is needed by the old Dockerfile (standalone version now)?

I may add later

File: docker/llm/finetune/qlora/cpu/docker/Dockerfile [hunk 1]
```diff
@@ -6,69 +6,67 @@ RUN apt-get update && \
...
     curl -fsSL https://apt.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS-2023.PUB | gpg --dearmor | tee /root/intel-oneapi-archive-keyring.gpg
 
-FROM mpioperator/intel as builder
+FROM debian:bullseye as builder
 
 ARG http_proxy
 ARG https_proxy
 ENV TZ=Asia/Shanghai
-ARG PIP_NO_CACHE_DIR=false
+ARG PIP_NO_CACHE_DIR=true
```
Comment (Uxito-Ada) [inline docker/llm/finetune/qlora/cpu/docker/Dockerfile:14]: why true here? false may save more storage

File: docker/llm/finetune/qlora/cpu/docker/Dockerfile [hunk 2]
```diff
@@ -6,69 +6,67 @@ RUN apt-get update && \
...
-# add bigdl-llm qlora cpu example
     cd /bigdl && \
     git clone https://github.com/intel-analytics/BigDL.git && \
     mv BigDL/python/llm/example/CPU/QLoRA-FineTuning/* . && \
-    rm -r BigDL
+    rm -r BigDL && \
+    chown -R mpiuser /bigdl
    
-# for docker directly run example
+# fro standalone
```
Comment (Uxito-Ada) [inline docker/llm/finetune/qlora/cpu/docker/Dockerfile:67]: typo

**Comment by Uxito-Ada** [comment]
please also update image names below:
https://github.com/intel-analytics/BigDL/blob/main/docker/llm/finetune/qlora/cpu/docker/README.md
https://github.com/intel-analytics/BigDL/blob/main/docker/llm/finetune/qlora/cpu/kubernetes/values.yaml

We can replace with two new names:
intelanalytics/bigdl-llm-finetune-qlora-cpu-standalone:2.5.0-SNAPSHOT
intelanalytics/bigdl-llm-finetune-qlora-cpu-k8s:2.5.0-SNAPSHOT

And please release according images to dockerhub after PR merged, tks.

File: docker/llm/finetune/qlora/cpu/docker/README.md [hunk 3]
```diff
@@ -13,13 +13,24 @@ docker pull intelanalytics/bigdl-llm-finetune-qlora-cpu:2.4.0-SNAPSHOT
...
+
+# For k8s
+export HTTP_PROXY=your_http_proxy
+export HTTPS_PROXY=your_https_proxy
+
+docker build \
+  --build-arg http_proxy=${HTTP_PROXY} \
+  --build-arg https_proxy=${HTTPS_PROXY} \
+  -t intelanalytics/bigdl-llm-finetune-qlora-cpu-k8s:2.5.0-SNAPSHOT \
   -f ./Dockerfile .
```
Comment (Uxito-Ada) [inline docker/llm/finetune/qlora/cpu/docker/README.md:34]: Dockerfile.k8s

Comment (Romanticoseu) [reply docker/llm/finetune/qlora/cpu/docker/README.md:34]: Ok, I got it.

**Comment by Uxito-Ada** [comment]
And image to pull: https://github.com/intel-analytics/BigDL/blob/70e2c9170238ceb6ed6b296bb0e5746c60dac58f/docker/llm/finetune/qlora/cpu/docker/README.md#1-prepare-docker-image

**Review by glorysdj — APPROVED** [review]
LGTM

**Pull request merged**